### PR TITLE
Stable: no comment

### PR DESCRIPTION
**Pull request by EndymionJkb**
Proof we need to finish the tests = this

### CODE REVIEW

File: pkg/pool-stable-phantom/contracts/StablePhantomPool.sol [hunk 1]
```diff
@@ -1167,6 +1169,8 @@ contract StablePhantomPool is
         uint256[] memory arguments,
         function(uint256, uint256) pure returns (uint256) mutation
     ) private pure {
+        InputHelpers.ensureInputLengthMatch(toMutate.length, arguments.length);
```
Comment (TomAFrench) [inline pkg/pool-stable-phantom/contracts/StablePhantomPool.sol:1172]: This just increases runtime cost for no reason. We shouldn't ever be passing in arrays which we don't _know_ are the same length.

**Review by TomAFrench — COMMENTED** [review]
oof, yeah this isn't nice.

File: pkg/pool-stable-phantom/contracts/StablePhantomPool.sol (hunk 1 above)
Comment (EndymionJkb) [reply pkg/pool-stable-phantom/contracts/StablePhantomPool.sol:1172]: I agree we *shouldn't*, except we just were :) I put it in to make sure there wasn't another issue. I can take it out, but we should make sure the tests cover this (at least indirectly). The fee amounts should eventually be wrong if we try different token combinations, at least before and after the BPT token.

Comment (TomAFrench) [reply pkg/pool-stable-phantom/contracts/StablePhantomPool.sol]: Yeah, to be clear I'm in favour of us having stuff which makes noise if we hit unexpected behaviour, just that we should remove these from the final build before we deploy. We could just make this a TODO to remove once tests are more comprehensive.

Comment (EndymionJkb) [reply pkg/pool-stable-phantom/contracts/StablePhantomPool.sol]: Well, I already removed it, hoping we could quietly merge and never speak of this again :)

I hesitate to make notes to do things "later," knowing that all too often, later never comes. I'd definitely keep it in your local copy while we're writing the tests, though.

**Review by TomAFrench — APPROVED** [review]

**Pull request merged**